Lecture_18/Homework_18.py

```diff
@@ -0,0 +1,36 @@
+class Bot:
+    def __init__(self, name):
+        self.name = name
+    def say_name(self, name):
```

Comment:
Aaah! here is the problem, method `say_name` waits `name` as required argument
```suggestion
    def say_name(self):
```